ui/src/views/sql-query-editor/components/state-filled.tsx

```diff
@@ -0,0 +1,239 @@
+import { Alert, Badge, Button, Input, Label, Select, Toolbar } from '@mergestat/blocks'
+import { ChevronLeftIcon, ChevronRightIcon, CircleCheckFilledIcon, ClipboardIcon, DownloadIcon, SearchIcon } from '@mergestat/icons'
+import cx from 'classnames'
+import { debounce } from 'lodash'
+import { useEffect, useState } from 'react'
+import { copy, filterByAllFields, paginate } from 'src/utils'
+import { EXPORT_FORMAT } from 'src/utils/constants'
+
+type QueryEditorFilledProps = {
+  rowLimit: number
+  rowLimitReached: boolean
+  data: Array<string | number | boolean>
+  children?: React.ReactNode
+}
+
+const QueryEditorFilled: React.FC<QueryEditorFilledProps> = ({ rowLimit, rowLimitReached, data }: QueryEditorFilledProps) => {
+  const [result, setResult] = useState<Array<string | number | boolean>>(data)
+  const [rows, setRows] = useState<number>(20)
+  const [page, setPage] = useState<number>(0)
+  const [total, setTotal] = useState<number>(data.length)
+  const [search, setSearch] = useState<string>('')
+  const [exportFormat, setExportFormat] = useState<string>(EXPORT_FORMAT.JSON)
+
+  const getData = (value: string | number | boolean) => {
+    if (value === null) {
+      return 'null'
+    }
+    if (value) {
+      if (typeof value === 'boolean') {
+        return value.toString()
+      } else if (value.constructor === ({}).constructor || value.constructor === ([]).constructor) {
+        return JSON.stringify(value)
+      }
+    }
+    return value
+  }
+
+  const getMax = () => {
+    const max = (page + 1) * rows
+    return max > total ? total : max
+  }
+
+  const isSpecial = (value: string | number | boolean) => {
+    return value === null || typeof value === 'boolean'
+  }
+
+  const getText = () => {
+    let text
+    if (exportFormat === EXPORT_FORMAT.JSON) {
+      text = JSON.stringify(data)
+    } else {
+      text = [
+        Object.keys(result[0]).join(','),
+        ...data.map(d => Object.values(d).map(getData).join(','))
+      ].join('\n')
+    }
+    return text
+  }
+
+  const copyToClipboard = () => {
+    copy(getText())
+  }
+
+  const exportData = () => {
+    const jsonString = `data:text/${exportFormat.toLowerCase()};chatset=utf-8,${encodeURIComponent(getText())}`
```

Comment:
```suggestion
    const jsonString = `data:text/${exportFormat.toLowerCase()};charset=utf-8,${encodeURIComponent(getText())}`
```

I'm guessing that's a typo?